dotnet/src/dotnetframework/GxClasses/Helpers/HttpHelper.cs

```diff
@@ -67,39 +67,22 @@ public class HttpHelper
 		public const string HttpPostMethod= "POST";
 		public const string HttpGetMethod = "GET";
 
+		const string OTP_USER_ACCESS_CODE_SENT = "400";
+		const string TFA_USER_MUST_VALIDATE = "410";
+		const string TOKEN_EXPIRED = "103";
+		const string ACCEPTED = "202";
+		const string FORBIDDEN = "403";
```

Comment:
These constants, even though the programming type is always `string`, they are of different semantic types.

I would recommend adding a prefix to each one, like `HTTP_STATUS_` or `GAM_CODE_` so that they don't get mixed up. You could also add constants for all the other http statuses that are written as literals in the code.